components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from "react";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import TextField from "@mui/material/TextField";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import styles from "./InviteAdminModal.module.css";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Icon from "../../assets/Icon";
+import { addAdminInvite } from "../../firebase/firestore/invite_admin";
+import { AdminInvite } from "../../types/schema";
+import { useForm } from "react-hook-form";
+import * as Yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import emailjs from "emailjs-com";
+
+export const InviteAdminModal: React.FunctionComponent = () => {
+  const [open, setOpen] = React.useState(false);
+  const [sent, setSent] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+    setSent(false);
+  };
+
+  const removeRow = (i) => {
+    unregister("invite." + i);
+    unregister("invite." + i);
+    setCount(count - 1);
+
+    setInvites(invites.filter((j) => j != i));
+  };
+
+  const theme = createTheme({
+    typography: {
+      fontFamily: "Avenir",
+    },
+    palette: {
+      primary: {
+        main: "#253C85",
+      },
+    },
+  });
+
+  const sendEmail = (templateParams) => {
+    emailjs
+      .send(
+        "service_o683bxk",
```

Comment:
these values within `email.send` could be re-used else where. Could you define them in a file called `constants.ts` and import them to your file (this way we can reuse them and it is good coding practice!)